viewer/src/components/LdTagInput.vue

```diff
@@ -58,6 +59,13 @@ export default class LdTagInput extends Vue {
       .filter(newSearch => !this.model.find(currentSearch => currentSearch.tag === newSearch.tag))
       .sort((a, b) => b.items.length - a.items.length);
   }
+
+  // Prevents the keyboard from opening on mobile when removing a tag
+  onClick(e: MouseEvent) {
+    const target = e.target;
+    if (target instanceof HTMLAnchorElement)
+      target.addEventListener("click", e => e.stopPropagation());
```

Comment:
```suggestion
      target.addEventListener("click", e => e.stopPropagation(), true);
```

It has a weird behaviour with keyboard navigation in qutebrowser: the browser switches to INPUT mode only when deleting a tag, despite not focusing on that field. This is happening at random and hints at a race-condition in the event capture.  __Edit: solved as proposed.__